clang/lib/CIR/CodeGen/CIRGenExprComplex.cpp

```diff
@@ -32,6 +46,16 @@ static const ComplexType *getComplexType(QualType type) {
   return cast<ComplexType>(cast<AtomicType>(type)->getValueType());
 }
 
+mlir::Value ComplexExprEmitter::emitLoadOfLValue(LValue lv,
+                                                 SourceLocation loc) {
+  assert(lv.isSimple() && "non-simple complex l-value?");
+  if (lv.getType()->isAtomicType())
+    cgf.cgm.errorNYI("emitLoadOfLValue with Atomic LV");
```

Comment:
```suggestion
    cgf.cgm.errorNYI(loc, "emitLoadOfLValue with Atomic LV");
```
This is very useful in the diagnostic produced.